packages/api-docs/src/index.js

```diff
@@ -73,8 +73,17 @@ async function generateApiForFiles(files) {
   return md;
 }
 
+// unix-like Operating systems requires quotation marks in it's string in the command line
```

Comment:
not quite true :grin:

the reality is that on a unix shell such as bash, placing quotation marks around strings on the command line stop their values from getting expanded - without these quotes on a bash shell including `**` in a path expression will expand to the first item it finds that's matching.

our command in the `package.json` wraps a path expression in `'` characters to ensure the `**` doesn't get expanded, thus allowing it to still be present in the `relDir` argument below.

bash does not pass on the `'` characters surrounding the command argument, as it recognises them as part of the argument - they get stripped away automatically before the command argument 

windows shells (i think both command.com or powershell) don't expand the `**`, so there the `'` characters aren't necessary.  additionally they don't understand wrapping arguments in `'` characters, so they pass those characters through